app/controllers/admin/order_cycles_controller.rb

```diff
@@ -70,6 +69,12 @@ def clone
       redirect_to main_app.admin_order_cycles_path, :notice => "Your order cycle #{@order_cycle.name} has been cloned."
     end
 
+    # Send notifications to all producers who are part of the order cycle
+    def notifications
+      @order_cycle = OrderCycle.find params[:id]
+      Delayed::Job.enqueue OrderCycleNotificationJob.new(@order_cycle)
```

Comment:
The usual pattern for displaying the result of an action like this would be to redirect back to the order cycles index page with a flash message like "Notification emails sent", rather than rendering a template.